### PR TITLE
Add info for Sensor Temperature

### PR DESCRIPTION
**Pull request by sinhaashish**
## Description
Fetches the sensor temperature using `SensorsTemperatures()` of 
`https://github.com/shirou/gopsutil`. 

## How to test this PR?
Run `hw-sensor-temp.go`

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [X] Documentation needed
- [ ] Unit tests needed
- [ ] Functional tests needed (If yes, add [mint](https://github.com/minio/mint) PR # here: )

### CODE REVIEW

File: cmd/endpoint.go [hunk 1]
```diff
@@ -744,3 +745,36 @@ func updateDomainIPs(endPoints set.StringSet) {
...
+
+// localEndpointsSensorTemp - returns ServerSensorTempInfo only for the
+// local endpoints from given list of endpoints
+func localEndpointsSensorTemp(endpoints EndpointList, r *http.Request) ServerSensorTemp {
+
+	var temps []host.TemperatureStat
+	scratchSpace := map[string]bool{}
+	for _, endpoint := range endpoints {
+		// Only proceed for local endpoints
+		if endpoint.IsLocal {
```
Comment (harshavardhana) [inline cmd/endpoint.go]: This is an incorrect code.. check https://github.com/minio/minio/pull/8191 and this should be incorporated in all the info PRs and all of this code should move to `admin-server-info.go`  - endpoint.go is only meat for command line parsing.

**Review by harshavardhana — CHANGES_REQUESTED** [review]
https://github.com/minio/minio/pull/8191 @poornas @kannappanr this PR is blocking changes being done by @sinhaashish - It would benefit if we can merge it and then let him rebase his work on top.

**Comment by kannappanr** [comment]
#8191 has been merged

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :x: [more...](#fed4e38c-3f28-4b1f-906c-2f369e23a7bb) |
| mint-xl.sh | :x: [more...](#eb4b1f4c-78f6-4587-88fc-0295b29ab5ee) |
| mint-large-bucket.sh | :x: [more...](#adfd1cf8-d0c1-4ddc-8584-dcfd0d4cdae8) |
| mint-compression-dist-xl.sh | :x: [more...](#d51353ce-69b1-422f-a12c-19ee489b72db) |
<a name="fed4e38c-3f28-4b1f-906c-2f369e23a7bb"></a>
### 8229-c3746ec/mint-compression-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.56:32369
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 8e5d725b2a18:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 6 minutes and 6 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-al12zc3eovosaa",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 366129,
  "error": "XMinioServerNotInitialized: Server not initialized, please try again.\n\tstatus code: 503, request id: 15C4EB9DD9ED0E00, host id: ",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```
<a name="d51353ce-69b1-422f-a12c-19ee489b72db"></a>
### 8229-c3746ec/mint-compression-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.60:31717
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 3e3510e2642c:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 1 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 43 seconds
(4/14) Running aws-sdk-ruby tests ... done in 4 seconds
(5/14) Running awscli tests ... done in 2 minutes and 6 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 52 seconds
(8/14) Running minio-dotnet tests ... done in 34 seconds
(9/14) Running minio-go tests ... done in 1 minutes and 0 seconds
(10/14) Running minio-java tests ... done in 1 minutes and 29 seconds
(11/14) Running minio-js tests ... done in 51 seconds
(12/14) Running minio-py tests ... FAILED in 2 minutes and 50 seconds
{
  "status": "FAIL",
  "duration": 102,
  "name": "minio-py:test_presigned_get_object_expiry_5sec",
  "message": "AccessDenied: message: Access Denied",
  "args": {
    "response.reason": "Forbidden",
    "presigned_get_object_url": "http://72.28.97.60:31717/minio-py-test-b63bc6dd-8213-489b-bfbc-162a5a87a3fb/d5e188af-1309-4e37-8958-f515681d4047?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=minio%2F20190916%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20190916T124453Z&X-Amz-Expires=5&X-Amz-SignedHeaders=host&X-Amz-Signature=bd85d23125b8dc6225c93b3ff61b13d2ec065d54c4377564b4fcad0afe72c953",
    "response._body": "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<Error><Code>AccessDenied</Code><Message>Request has expired</Message><Key>d5e188af-1309-4e37-8958-f515681d4047</Key><BucketName>minio-py-test-b63bc6dd-8213-489b-bfbc-162a5a87a3fb</BucketName><Resource>/minio-py-test-b63bc6dd-8213-489b-bfbc-162a5a87a3fb/d5e188af-1309-4e37-8958-f515681d4047</Resource><RequestId>15C4EBDB9912BDE1</RequestId><HostId>2d6aeb2d-7096-48df-87a8-f2e81ffcc25b</HostId></Error>",
    "response.headers": "{\"_container\": {\"accept-ranges\": [\"Accept-Ranges\", \"bytes\"], \"content-length\": [\"Content-Length\", \"440\"], \"content-security-policy\": [\"Content-Security-Policy\", \"block-all-mixed-content\"], \"content-type\": [\"Content-Type\", \"application/xml\"], \"server\": [\"Server\", \"MinIO/DEVELOPMENT.2019-09-16T12-21-30Z\"], \"vary\": [\"Vary\", \"Origin\"], \"x-amz-request-id\": [\"X-Amz-Request-Id\", \"15C4EBDB9912BDE1\"], \"x-xss-protection\": [\"X-Xss-Protection\", \"1; mode=block\"], \"date\": [\"Date\", \"Mon, 16 Sep 2019 12:44:58 GMT\"]}}",
    "object_name": "d5e188af-1309-4e37-8958-f515681d4047",
    "response.status": 403,
    "bucket_name": "minio-py-test-b63bc6dd-8213-489b-bfbc-162a5a87a3fb"
  },
  "error": "Traceback (most recent call last):\n  File \"/mint/run/core/minio-py/tests.py\", line 1341, in test_presigned_get_object_expiry_5sec\n    object_name).get_exception()\nminio.error.AccessDenied: AccessDenied: message: Access Denied\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/mint/run/core/minio-py/tests.py\", line 2036, in main\n    test_presigned_get_object_expiry_5sec(client, log_output)\n  File \"/mint/run/core/minio-py/tests.py\", line 1355, in test_presigned_get_object_expiry_5sec\n    raise Exception(err)\nException: AccessDenied: message: Access Denied\n",
  "function": "presigned_get_object(bucket_name, object_name, expires, response_headers, request_date)"
}

Executed 11 out of 14 tests successfully.
```
<a name="adfd1cf8-d0c1-4ddc-8584-dcfd0d4cdae8"></a>
### 8229-c3746ec/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.61:31949
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp fb37b2654dd9:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 3 seconds
(2/14) Running aws-sdk-java tests ... done in 2 seconds
(3/14) Running aws-sdk-php tests ... FAILED in 5 minutes and 50 seconds
{
  "name": "aws-sdk-php",
  "function": "getBucketPolicy ( array $params = [] )",
  "args": {
    "Bucket": "aws-sdk-php-5d7f815345160"
  },
  "duration": "41162",
  "status": "FAIL",
  "error": "Error executing \"PutObject\" on \"http://72.28.97.61:31949/aws-sdk-php-5d7f815345160/test-anon\"; AWS HTTP error: Client error: `PUT http://72.28.97.61:31949/aws-sdk-php-5d7f815345160/test-anon` resulted in a `400 Bad Request` response:\n<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<Error><Code>IncompleteBody</Code><Message>You did not provide the number of byte (truncated...)\n IncompleteBody (client): You did not provide the number of bytes specified by the Content-Length HTTP header. - <?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<Error><Code>IncompleteBody</Code><Message>You did not provide the number of bytes specified by the Content-Length HTTP header.</Message><Key>test-anon</Key><BucketName>aws-sdk-php-5d7f815345160</BucketName><Resource>/aws-sdk-php-5d7f815345160/test-anon</Resource><RequestId>15C4EB9A1A094E8E</RequestId><HostId>17ee6eea-7698-47a5-b445-cdcbd0ef9c5f</HostId></Error>"
}

Executed 2 out of 14 tests successfully.
```
<a name="eb4b1f4c-78f6-4587-88fc-0295b29ab5ee"></a>
### 8229-c3746ec/mint-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.59:31242
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 7e4268779676:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 6 minutes and 6 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-3celoddyydo9s6",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 365815,
  "error": "XMinioServerNotInitialized: Server not initialized, please try again.\n\tstatus code: 503, request id: 15C4EB9CA346BC33, host id: ",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :x: [more...](#90b8b2bf-ab4f-4150-b039-dc533f660f1a) |
| mint-large-bucket.sh | :x: [more...](#bfc1ca98-19a0-4a75-a24c-d3d4ed53d7c6) |
| mint-gateway-nas.sh | :x: [more...](#aedfd9cc-9575-4ddd-8ad5-2b5f82e592cd) |
<a name="aedfd9cc-9575-4ddd-8ad5-2b5f82e592cd"></a>
### 8229-41a10bd/mint-gateway-nas.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.51:31313
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp f576734983e4:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 1 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 40 seconds
(4/14) Running aws-sdk-ruby tests ... done in 2 seconds
(5/14) Running awscli tests ... done in 1 minutes and 51 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 49 seconds
(8/14) Running minio-dotnet tests ... done in 23 seconds
(9/14) Running minio-go tests ... done in 48 seconds
(10/14) Running minio-java tests ... done in 23 seconds
(11/14) Running minio-js tests ... FAILED in 3 seconds
{
  "name": "minio-js",
  "function": "\"after all\" hook in \"functional tests\"",
  "duration": 6,
  "status": "FAIL",
  "error": "S3Error: The bucket you tried to delete is not empty at Object.parseError (node_modules/minio/dist/main/xml-parsers.js:83:11) at /mint/run/core/minio-js/node_modules/minio/dist/main/transformers.js:153:22 at DestroyableTransform._flush (node_modules/minio/dist/main/transformers.js:77:10) at DestroyableTransform.prefinish (node_modules/readable-stream/lib/_stream_transform.js:129:10) at prefinish (node_modules/readable-stream/lib/_stream_writable.js:607:14) at finishMaybe (node_modules/readable-stream/lib/_stream_writable.js:616:5) at endWritable (node_modules/readable-stream/lib/_stream_writable.js:629:3) at DestroyableTransform.Writable.end (node_modules/readable-stream/lib/_stream_writable.js:567:22) at IncomingMessage.onend (_stream_readable.js:514:10) at endReadableNT (_stream_readable.js:978:12) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)"
}

Executed 10 out of 14 tests successfully.
```
<a name="90b8b2bf-ab4f-4150-b039-dc533f660f1a"></a>
### 8229-41a10bd/mint-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.59:32162
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 6911b2ee4c18:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 0 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 42 seconds
(4/14) Running aws-sdk-ruby tests ... done in 1 seconds
(5/14) Running awscli tests ... done in 1 minutes and 54 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 50 seconds
(8/14) Running minio-dotnet tests ... done in 25 seconds
(9/14) Running minio-go tests ... done in 59 seconds
(10/14) Running minio-java tests ... done in 19 seconds
(11/14) Running minio-js tests ... FAILED in 16 seconds
{
  "name": "minio-js",
  "function": "fPutObject(bucketName, objectName, filePath, metaData)",
  "args": "bucketName:minio-js-test-5ae60000-44a1-426b-a932-30a99ff6337c, objectName:datafile-65-MB, filePath:/tmp/datafile-65-MB",
  "duration": 2614,
  "status": "FAIL",
  "error": "Error: read ECONNRESET at exports._errnoException (util.js:1020:11) at TCP.onread (net.js:580:26)"
}

Executed 10 out of 14 tests successfully.
```
<a name="bfc1ca98-19a0-4a75-a24c-d3d4ed53d7c6"></a>
### 8229-41a10bd/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.59:30285
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 1defd6b914c3:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 3 seconds
(2/14) Running aws-sdk-java tests ... done in 2 seconds
(3/14) Running aws-sdk-php tests ... done in 52 seconds
(4/14) Running aws-sdk-ruby tests ... done in 17 seconds
(5/14) Running awscli tests ... done in 2 minutes and 25 seconds
(6/14) Running healthcheck tests ... done in 1 seconds
(7/14) Running mc tests ... done in 1 minutes and 19 seconds
(8/14) Running minio-dotnet tests ... done in 1 minutes and 23 seconds
(9/14) Running minio-go tests ... FAILED in 4 minutes and 6 seconds
{
  "args": {},
  "duration": 97167,
  "error": "The specified key does not exist.",
  "function": "testStorageClassMetadataCopyObject()",
  "message": "Stat failed",
  "name": "minio-go: testStorageClassMetadataCopyObject",
  "status": "FAIL"
}

Executed 8 out of 14 tests successfully.
```

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :x: [more...](#cc3216bc-0e09-4ab9-acd4-f43581910d1a) |
| mint-compression-fs.sh | :x: [more...](#f08aa380-7d48-4d1c-8b7b-5dcb529ec1c7) |
| mint-gateway-nas.sh | :x: [more...](#85c240ea-89c5-4247-bd34-6dddac9f5ed0) |
<a name="85c240ea-89c5-4247-bd34-6dddac9f5ed0"></a>
### 8229-2768c0d/mint-gateway-nas.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:31255
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp d2e86e8bf13a:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 0 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 42 seconds
(4/14) Running aws-sdk-ruby tests ... done in 2 seconds
(5/14) Running awscli tests ... done in 1 minutes and 57 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 47 seconds
(8/14) Running minio-dotnet tests ... done in 24 seconds
(9/14) Running minio-go tests ... done in 48 seconds
(10/14) Running minio-java tests ... done in 1 minutes and 31 seconds
(11/14) Running minio-js tests ... done in 30 seconds
(12/14) Running minio-py tests ... FAILED in 3 minutes and 43 seconds
{
  "duration": 63,
  "error": "Traceback (most recent call last):\n  File \"/mint/run/core/minio-py/tests.py\", line 1341, in test_presigned_get_object_expiry_5sec\n    object_name).get_exception()\nminio.error.AccessDenied: AccessDenied: message: Access Denied\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/mint/run/core/minio-py/tests.py\", line 2036, in main\n    test_presigned_get_object_expiry_5sec(client, log_output)\n  File \"/mint/run/core/minio-py/tests.py\", line 1355, in test_presigned_get_object_expiry_5sec\n    raise Exception(err)\nException: AccessDenied: message: Access Denied\n",
  "name": "minio-py:test_presigned_get_object_expiry_5sec",
  "function": "presigned_get_object(bucket_name, object_name, expires, response_headers, request_date)",
  "args": {
    "bucket_name": "minio-py-test-c400b4b2-968e-4de9-be5e-059580ca1366",
    "presigned_get_object_url": "http://72.28.97.55:31255/minio-py-test-c400b4b2-968e-4de9-be5e-059580ca1366/611d4625-3a70-4d07-a64c-a6c8e329c8b3?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=minio%2F20190916%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20190916T183930Z&X-Amz-Expires=5&X-Amz-SignedHeaders=host&X-Amz-Signature=aa800eaf53933915f0ac786883e28aa43b222886f13069b202be2db338250f42",
    "response._body": "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<Error><Code>AccessDenied</Code><Message>Request has expired</Message><Key>611d4625-3a70-4d07-a64c-a6c8e329c8b3</Key><BucketName>minio-py-test-c400b4b2-968e-4de9-be5e-059580ca1366</BucketName><Resource>/minio-py-test-c400b4b2-968e-4de9-be5e-059580ca1366/611d4625-3a70-4d07-a64c-a6c8e329c8b3</Resource><RequestId>15C4FF3589593A03</RequestId><HostId></HostId></Error>",
    "response.reason": "Forbidden",
    "response.status": 403,
    "response.headers": "{\"_container\": {\"accept-ranges\": [\"Accept-Ranges\", \"bytes\"], \"content-length\": [\"Content-Length\", \"404\"], \"content-security-policy\": [\"Content-Security-Policy\", \"block-all-mixed-content\"], \"content-type\": [\"Content-Type\", \"application/xml\"], \"server\": [\"Server\", \"MinIO/DEVELOPMENT.2019-09-16T18-16-14Z\"], \"vary\": [\"Vary\", \"Origin\"], \"x-amz-request-id\": [\"X-Amz-Request-Id\", \"15C4FF3589593A03\"], \"x-xss-protection\": [\"X-Xss-Protection\", \"1; mode=block\"], \"date\": [\"Date\", \"Mon, 16 Sep 2019 18:39:35 GMT\"]}}",
    "object_name": "611d4625-3a70-4d07-a64c-a6c8e329c8b3"
  },
  "message": "AccessDenied: message: Access Denied",
  "status": "FAIL"
}

Executed 11 out of 14 tests successfully.
```
<a name="f08aa380-7d48-4d1c-8b7b-5dcb529ec1c7"></a>
### 8229-2768c0d/mint-compression-fs.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.61:30324
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 30534e816abc:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 0 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 42 seconds
(4/14) Running aws-sdk-ruby tests ... done in 2 seconds
(5/14) Running awscli tests ... done in 1 minutes and 52 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 40 seconds
(8/14) Running minio-dotnet tests ... done in 26 seconds
(9/14) Running minio-go tests ... done in 47 seconds
(10/14) Running minio-java tests ... done in 1 minutes and 31 seconds
(11/14) Running minio-js tests ... done in 45 seconds
(12/14) Running minio-py tests ... FAILED in 3 minutes and 52 seconds
{
  "message": "AccessDenied: message: Access Denied",
  "error": "Traceback (most recent call last):\n  File \"/mint/run/core/minio-py/tests.py\", line 1341, in test_presigned_get_object_expiry_5sec\n    object_name).get_exception()\nminio.error.AccessDenied: AccessDenied: message: Access Denied\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/mint/run/core/minio-py/tests.py\", line 2036, in main\n    test_presigned_get_object_expiry_5sec(client, log_output)\n  File \"/mint/run/core/minio-py/tests.py\", line 1355, in test_presigned_get_object_expiry_5sec\n    raise Exception(err)\nException: AccessDenied: message: Access Denied\n",
  "function": "presigned_get_object(bucket_name, object_name, expires, response_headers, request_date)",
  "duration": 73,
  "name": "minio-py:test_presigned_get_object_expiry_5sec",
  "args": {
    "object_name": "aa15107c-924a-4d55-84e9-fd170f4d786a",
    "response.reason": "Forbidden",
    "presigned_get_object_url": "http://72.28.97.61:30324/minio-py-test-63d5b974-9a06-44f3-8894-4f6c6aa8cc6c/aa15107c-924a-4d55-84e9-fd170f4d786a?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=minio%2F20190916%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20190916T183943Z&X-Amz-Expires=5&X-Amz-SignedHeaders=host&X-Amz-Signature=a73f872441dd759e358afe4463b643f6693bf6f9b5de45f8070e7d841b8650f6",
    "response._body": "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<Error><Code>AccessDenied</Code><Message>Request has expired</Message><Key>aa15107c-924a-4d55-84e9-fd170f4d786a</Key><BucketName>minio-py-test-63d5b974-9a06-44f3-8894-4f6c6aa8cc6c</BucketName><Resource>/minio-py-test-63d5b974-9a06-44f3-8894-4f6c6aa8cc6c/aa15107c-924a-4d55-84e9-fd170f4d786a</Resource><RequestId>15C4FF3891CFF742</RequestId><HostId>7ebab8c6-1f64-43da-988f-681338461bbd</HostId></Error>",
    "response.status": 403,
    "bucket_name": "minio-py-test-63d5b974-9a06-44f3-8894-4f6c6aa8cc6c",
    "response.headers": "{\"_container\": {\"accept-ranges\": [\"Accept-Ranges\", \"bytes\"], \"content-length\": [\"Content-Length\", \"440\"], \"content-security-policy\": [\"Content-Security-Policy\", \"block-all-mixed-content\"], \"content-type\": [\"Content-Type\", \"application/xml\"], \"server\": [\"Server\", \"MinIO/DEVELOPMENT.2019-09-16T18-16-14Z\"], \"vary\": [\"Vary\", \"Origin\"], \"x-amz-request-id\": [\"X-Amz-Request-Id\", \"15C4FF3891CFF742\"], \"x-xss-protection\": [\"X-Xss-Protection\", \"1; mode=block\"], \"date\": [\"Date\", \"Mon, 16 Sep 2019 18:39:48 GMT\"]}}"
  },
  "status": "FAIL"
}

Executed 11 out of 14 tests successfully.
```
<a name="cc3216bc-0e09-4ab9-acd4-f43581910d1a"></a>
### 8229-2768c0d/mint-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.53:30272
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 83c8a74b6231:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 6 minutes and 4 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-ohvj5g1mpbozz1",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 363970,
  "error": "XMinioServerNotInitialized: Server not initialized, please try again.\n\tstatus code: 503, request id: 15C4FEF76CE0BF4A, host id: ",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```

**Comment by sinhaashish** [comment]
Closing as its not required now

**Pull request closed without merging**